expr/functions.go

```diff
@@ -67,6 +68,11 @@ var allFns = map[string]struct {
 	"Time.fromMicroseconds": {1, 1, timeFromUsec},
 	"Time.fromNanoseconds":  {1, 1, timeFromNsec},
 	"Time.trunc":            {2, 2, timeTrunc},
+
+	"typeof":     {1, 1, typeOf},
+	"iserr":      {1, 1, isErr},
```

Comment:
@philrz and I talked about function names and I think we need a group consensus around naming conventions.  The current name all are package-style but we don't have packages so that seems wrong.  I'm not sure about lower camelCase.  It seems a little arbitrary.  Also for a interactive query language where a user often type lots of stuff rapid fire, getting capitalization right seems like a mistake.  I guess I am arguing for snake case where underscores are used only for long names that are less often types.  How about I leave this for now and we do a PR soon on whatever we decide.   Issue #1426